docs/sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,59 @@
 Version history
 ===============
 
+5.4.0 (2017 March 21)
+---------------------
+
+File format improvements:
+
+* DICOM
+   - added support for DICOMDIR files, which allow multiple DICOM files in a single 
+     directory to be opened as a single dataset
+* Nikon NIS-Elements ND2
+   - performance improvements based on reading chunkmap. Processing of the chunkmap 
+     can be disabled via the MetadataOptions API using the boolean option 
+     ``nativend2.chunkmap``. For ImageJ users this option can be accessed via a checkbox 
+     in the Nikon ND2 of the Bio-Formats configuration dialog 
+     ``Plugins > Bio-Formats > Bio-Formats Plugins Configuration should`` (thanks to Christian Sachs)
```

Comment:
@hflynn we will need an entry https://www.openmicroscopy.org/site/about/ome-contributors
since you are currently editing the page